<robot name="test_platform">
    
    <link name="base_footprint" />
    <link name="base_link" />
    <link name="gps" />
    <link name="imu" />
    <link name="camera" />
    <link name="lidar" />
    
    <joint name="groundToDriveWheelMidpoint" type="fixed">
        <parent link="base_footprint"/>
        <child link="base_link"/>
        <origin xyz="0 0 0.127" rpy="0 0 0"/>
        <axis xyz="0 0 0"/>
    </joint>
    
    <joint name="gps" type="fixed">
        <parent link="base_link"/>
        <child link="gps"/>
        <origin xyz="0 0.285 0.333" rpy="0 0 0"/>
        <axis xyz="0 0 0"/>
    </joint>
    
    <joint name="imu" type="fixed">
        <parent link="base_link"/>
        <child link="imu"/>
        <origin xyz="0 -0.11 1.083" rpy="0 0 0"/>
        <axis xyz="0 0 0"/>
    </joint>
    
    <joint name="camera" type="fixed">
        <parent link="base_link"/>
        <child link="camera"/>
        <origin xyz="0 -0.11 1.193" rpy="-0.3491 0 0"/>
        <axis xyz="0 0 0"/>
    </joint>
    
    <joint name="lidar" type="fixed">
        <parent link="base_link"/>
        <child link="lidar"/>
        <origin xyz="0 0.52 0.243" rpy="0 0 0"/>
        <axis xyz="0 0 0"/>
    </joint>
</robot>
